app/src/main/java/org/apache/fineract/ui/adapters/CustomerAdapter.java

```diff
@@ -115,12 +120,16 @@ public ViewHolder(View v) {
 
         @Override
         public void onClick(View v) {
-            onItemClickListener.onItemClick(v, getAdapterPosition());
+            if (onItemClickListener != null) {
```

Comment:
Nope again, It's can't be null. onItemClickListener should be `lateinit`

Reply:
It can be now, this same adapter is being used in the draft customers section where the draft customer cannot be clicked